helix-tui/src/buffer.rs

```diff
@@ -266,11 +266,30 @@ impl Buffer {
         width: usize,
         style: Style,
     ) -> (u16, u16)
+    where
+        S: AsRef<str>,
+    {
+        self.set_string_truncated(x, y, string, width, style, false)
+    }
+
+    /// Print at most the first `width` characters of a string if enough space is available
+    /// until the end of the line. If `markend` is true appends a `…` at the end of
+    /// truncated lines.
+    pub fn set_string_truncated<S>(
+        &mut self,
+        x: u16,
+        y: u16,
+        string: S,
+        width: usize,
+        style: Style,
+        markend: bool,
```

Comment:
Call this `ellipsis`

Reply:
I was looking for a better name :) Done.